sdk/tests/deqp/functional/gles3/es3fFboTestCase.js

```diff
@@ -135,7 +135,7 @@ var DE_ASSERT = function(x) {
             /** @const @type {goog.NumberArray} */ var supportedSampleCounts = es3fFboTestCase.querySampleCounts(sizedFormat);
             var supported = Array.prototype.slice.call(supportedSampleCounts);
             if (supported.indexOf(numSamples) == -1) {
-                if (numSamples >= 2) {
+                if (minSampleCount == 0 || numSamples > gl.getParameter(gl.MAX_SAMPLES)) {
```

Comment:
Reviewing this change and studying the spec it looks correct to me, but I'd like @zhenyao to review it too and ask @phaulos to please take a look too.